Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "foxbox"
 version = "0.1.0"
 authors = ["fabrice <fabrice@desre.org>"]
 build = "build.rs"
+exclude = [ "config/**" ]
```

Comment:
The openzwave library writes in this directory, we don't want to retrigger a build after each run. ;)